vendor/knative.dev/pkg/test/zipkin/util.go

```diff
@@ -164,9 +164,9 @@ func jsonTrace(traceID string) ([]model.SpanModel, error) {
 	var empty []model.SpanModel
 
 	// Check if zipkin port forwarding is setup correctly
-	if err := CheckZipkinPortAvailability(); err == nil {
-		return empty, err
-	}
+	// if err := CheckZipkinPortAvailability(); err == nil {
```

Comment:
This is just temporary, the real fix is https://github.com/knative/pkg/pull/693.